ios/Classes/SwiftDatadomePlugin.swift

```diff
@@ -56,6 +56,10 @@ public class SwiftDatadomePlugin: NSObject, FlutterPlugin {
             return input as? Data
         }
         
+        if let input = input as? FlutterStandardTypedData {
+            return input.data
+        }
+        
```

Comment:
nitpick:
`input` here shadows the initial `input` variable.
```suggestion
        if let typedData = input as? FlutterStandardTypedData {
    return typedData.data
}
```

Reply:
It's normal in Swift (and encouraged as best practice), it's a local override that is "killed" at the end of the `if` context window